assets/javascripts/modules/responsiveInlineDetails.js

```diff
@@ -0,0 +1,30 @@
+module.exports = function() {
+
+  $('[data-responsive-inline-details]').each(function() {
+    var $detailsHolder,
+    $detailsTargets,
+    $detailsButton;
```

Comment:
It's a micro optimisation, all the variables declarations are processed at once rather than one at a time. I guess it depends on whether the js compiler is clever enough to handle this on its own or not. 

The optimisation effect is negligible if non existent (depending on the js compiler handling of multiple var declarations) so if people feel that the code is more readable with multiple declarations then I'm happy to change this